### ISSUE TITLE
No Symbols on the text field

### ISSUE THREAD

**Issue by SoftzillaAT** [iam4x/pokemongo-webspoof#235]
Hello,

i can't set custom coordinates through the text fields lat and lng, because i am not able to put in a minus sign or a dot sign. Has there anybody a workaround? There is also no copy and paste functionality on the text fields. Has anyone the same problem?

**Comment by MKesenheimer** [comment]
Yes, I have the same problem, and I would like to have an workaround, too.
